apps/admin/backend/src/store.ts

```diff
@@ -1835,17 +1835,27 @@ export class Store {
   /**
    * Gets write-in tallies specifically for tabulation, filtered and and
    * grouped by cast vote record attributes.
+   *
+   * When using write-in tallies to hydrate tally results, unadjudicated and
+   * unmarked write-ins should not be included because they are not associated
+   * with an original or adjudicated mark. When using write-in tallies as a
+   * summary of write-in adjudication, they should be included because they
+   * exist in the adjudication flow just as marked write-ins do. The behavior
+   * can be controlled with the `includeUnadjudicatedAndInvalidUnmarkedWriteIns`
+   * flag.
```

Comment:
Should "unadjudicated and unmarked write-ins" in this comment specifically read "unadjudicated and *invalid* unmarked write-ins"?

Even with that change, there's a bit of ambiguity as this could mean "unadjudicated _marked_ write-ins and invalid _unmarked_ write-ins". I think "unmarked write-ins that haven't been adjudicated or are invalid" might be clearer

Reply:
Sounds good, change incorporated.

I can see someone making the same complaint about the variable name, but I think it reads a bit clearer as a variable name, so I am inclined to leave it.